app/jenkins-for-jira-ui/src/common/analytics/analytics-events.ts

```diff
@@ -41,4 +41,9 @@ export enum AnalyticsTrackEventsEnum {
 	UpdatedServerErrorName = 'updatedServerErrorName'
 }
 
-export enum AnalyticsOperationalEventsEnum {}
+export enum AnalyticsOperationalEventsEnum {
+	GetServerSuccessManageConnectionName = 'getServerManageSuccessConnection',
+	GetServerErrorManageConnectionName = 'getServerErrorManageConnection',
+	DisconnectSuccessServerManageConnectionName = 'disconnectedServerSuccessManageConnection',
```

Comment:
\nit Just to make sure the names are similar.
`DisconnectServerSuccessManageConnectionName`